src/tiles/item-image.ts

```diff
@@ -86,6 +88,7 @@ export class ItemImage extends LitElement {
       this.itemImageWaveform.naturalWidth /
       this.itemImageWaveform.naturalHeight;
     if (aspectRatio === 4) {
+      this.box.classList.add(this.randomGradient);
```

Comment:
Instead of manipulating classes directly, I would recommend setting a state property and change your classes based on the state. ie:
```ts
@state() isWaveform: boolean = false;
...
if (aspectRatio === 4) this.isWaveform = true;
...
<div class="box ${this.isWaveform ? this.randomGradient : ''}">
```
Similar comment for the line just below this line.